lib/msal-browser/src/navigation/NavigationClient.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { INavigationClient } from "./INavigationClient";
+import { NavigationOptions } from "./NavigationOptions";
+
+export class NavigationClient implements INavigationClient {
+    /**
+     * Navigates to other pages within the same web application
+     * @param url 
+     * @param options 
+     */
+    navigateInternal(url: string, options: NavigationOptions): Promise<boolean>{
+        if (options.noHistory) {
+            window.location.replace(url);
+        } else {
+            window.location.assign(url);
+        }
+
+        return new Promise((resolve) => {
+            setTimeout(() => {
+                resolve(true);
+            }, options.timeout);
+        });
```

Comment:
Seems like duplicated code here and below can we clean this up to call a helper?

Reply:
Done